run-tests.pl

```diff
@@ -56,6 +57,10 @@
 
 our $BIND_HOST = "localhost";
 
+# a unique ID for this test run. It is used in some tests to create user IDs
+# and the like.
+our $TEST_RUN_ID = time2str( '%Y%m%d%H%M%S', time() );
```

Comment:
Since we don't directly depend on `Date::Format` and it isn't core, it might be better to stick to just the core `POSIX` instead.

    use POSIX qw( strftime );

    our $TEST_RUN_ID = strftime( '%Y%m%d%H%M%S', localtime() );

Also might be nice to give a nod to ISO8601 and put a `T` in there, so it's easier to visually inspect where the actual time part of that sits, since it would be more human-readable:

    '%Y%m%dT%H%M%S'

Reply:
both good points; done.